contrib/kafka/Readme.md

```diff
@@ -0,0 +1,103 @@
+# Knative KafkaEventSource
+
+This deploys an Operator that can instantiate Knative Event Sources which subscribe to a Kafka Topic.
+
+## EventSource
+
+The image that receives messages from Kafka (Receive Adaptor) is available at `docker.io/sjwoodman/kafkaeventsource`.
```

Comment:
Is there an official Dockerhub account for Knative that we can use instead of your personal account?

Reply:
Not that I know of but very happy to move it if anyone can suggest somewhere suitable